core.osgi/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/osgi/commands/RepairTableStatusCommand.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecchronos.core.osgi.commands;
+
+import com.datastax.driver.core.Host;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.RepairJobView;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.RepairScheduler;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.state.RepairStateSnapshot;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.state.VnodeRepairState;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.state.VnodeRepairStates;
+import com.ericsson.bss.cassandra.ecchronos.core.utils.LongTokenRange;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableSet;
+import org.apache.karaf.shell.api.action.Action;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.Completion;
+import org.apache.karaf.shell.api.action.Option;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.apache.karaf.shell.support.CommandException;
+import org.apache.karaf.shell.support.completers.StringsCompleter;
+import org.apache.karaf.shell.support.table.ShellTable;
+
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+@Service
+@Command(scope = "repair", name = "table-status", description = "Give the current repair status for the given table")
+public class RepairTableStatusCommand implements Action
+{
+    private static final String SORT_RANGE = "RANGE";
+    private static final String SORT_REPAIRED_AT = "REPAIRED_AT";
+
+    @Reference
+    private RepairScheduler myRepairSchedulerService;
+
+    @Option(name = "-t", aliases = "table_reference", description = "The table reference in format <keyspace>.<table>", required = true, multiValued = false)
+    @Completion(TableReferenceCompleter.class)
+    String myTableRef;
+
+    @Option(name = "-s", aliases = {"--sort"}, description = "Sort output based on " + SORT_RANGE + "/"  + SORT_REPAIRED_AT, required = false, multiValued = false)
```

Comment:
How come this alias is different than the rest?

Reply:
remove { and }